common/flatpak-dir.h

```diff
@@ -670,6 +670,23 @@ gboolean   flatpak_dir_list_remote_refs (FlatpakDir   *self,
                                          GHashTable  **refs,
                                          GCancellable *cancellable,
                                          GError      **error);
+gboolean   flatpak_dir_list_local_remote_refs (FlatpakDir   *self,
+                                               const char   *remote,
+                                               GHashTable  **refs,
+                                               GCancellable *cancellable,
+                                               GError      **error);
+gboolean flatpak_dir_fetch_remote_repo_metadata (FlatpakDir    *self,
+                                                 const char    *remote_name,
+                                                 GCancellable  *cancellable,
+                                                 GError       **error);
+char *   flatpak_dir_fetch_remote_title (FlatpakDir   *self,
+                                         const char   *remote,
+                                         GCancellable *cancellable,
+                                         GError      **error);
+char *   flatpak_dir_fetch_remote_default_branch (FlatpakDir   *self,
+                                                  const char   *remote,
+                                                  GCancellable *cancellable,
+                                                  GError      **error);
```

Comment:
Whats with all these extra functions?

Reply:
Possibly an error introduced by resolving rebase conflicts. Thanks for pointing it out - I'll look into it.